Makefile

```diff
@@ -6,8 +6,14 @@ HANDLER_IMAGE_TAG ?= latest
 HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME):$(HANDLER_IMAGE_TAG)
 HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
 
-GINKGO_EXTRA_ARGS ?=
-GINKGO_ARGS ?= -v -r --randomizeAllSpecs --randomizeSuites --race --trace $(GINKGO_EXTRA_ARGS)
+UNIT_TEST_EXTRA_ARGS ?=
+UNIT_TEST_ARGS ?= -v -r --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_EXTRA_ARGS)
+ifdef UNIT_TEST_FOCUS
+	UNIT_TEST_ARGS += --focus $(UNIT_TEST_FOCUS)
+endif
+ifdef UNIT_TEST_SKIP
+	UNIT_TEST_ARGS += --skip $(UNIT_TEST_SKIP)
```

Comment:
this look familiar, except in the other code i saw, you had `+={space space}--...`, so there are spaces between flags

Reply:
Organized it a little, let me check if is good now.

Reply:
If I call focus and skip now, does it work? Will it include an empty space between the two?